### PR TITLE
feat: Adding java example for Charts embedding

### PR DESCRIPTION
**Pull request by jarjee**
CHARTS-2177

Charts Embedding with Verified Signature auth samples for Java.

### Java signed payload example
java signed payload example working

### CODE REVIEW

**Comment by shaketbaby** [comment]
Codes look good. I think we should remove `.classpath`, `.project` and `.settings` directories as these are specific to Eclipse (The last time I used Eclipse was about a decade or two ago, so I might be wrong). Maven's `pom.xml` file should be all that's required to get the example working.

File: java/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    - `~REPLACE~EMBEDDING_SIGNING_KEY` with the Embedding Signing key you obtained above
+
+4. Open the *src/main/resources/www/index.html* file (client-side code), and replace:
+    - `~REPLACE~CHART_ID` with the value of the *id* parameter from the IFRAME snippet you copied from Charts
+
+5. Run `mvn install` within *charts-embedding*
+
+6. Launch the application by running `java -jar target/charts-embedding-1.0-with-deps.jar`
+
+7. You should see the chart embedded within the sample page on [http://localhost:4567](http://localhost:4567).
```
Comment (shaketbaby) [inline java/README.md]: Which port do we use in other examples? Should we use the same port across all examples?

Comment (tomhollander) [reply java/README.md]: The .NET one is using port 50987, which was automatically generated by the tooling (not sure if it's a standard development port, or if it were randomly chosen). I don't think it really matters if the ports for different languages are different - in fact arguably there's value in them being different so you can run multiple simultaneously if you're into that kind of thing.

File: java/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+   *Embedding Options* from the *...* menu. Make sure that embedding is enabled for this data source,
+   using either authentication mode.
+
+4. Find the chart you want to embed, click the *...* menu and select Embed Chart.
+
+5. Select the *Verified Signature* tab and turn on embedding.
+
+6. Copy the IFRAME embed code from this dialog and keep it handy
+
+7. Get the Embedding Signing Key from the Charts Admin page. If you are not a Charts Admin you
```
Comment (tomhollander) [inline java/README.md]: I know this was a copy-paste from my examples, but this page now exists and is called "Admin Settings". Can you please correct it here and in the other two sample readmes?

Comment (jarjee) [reply java/README.md]: Sure, will do

**Comment by jarjee** [comment]
Now purged the eclipse files. I actually ended up generating them because of VSCode, but I've confirmed that eclipse can still open the project without those files.

**Pull request merged**